src/data/components/Rating/Rating.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+function Rating(props) {
+
+    let {children} = props;
+
+    let roundedChildren = Math.round(children);
+
+    switch(roundedChildren){
+        case 0:
+            children = '☆☆☆☆☆';
+            break;
+        case 1:
+            children = '★☆☆☆☆'
+            break;
+        case 2:
+            children = '★★☆☆☆'
+            break;
+        case 3:
+            children = '★★★☆☆';
+            break;
+        case 4:
+            children = '★★★★☆'
+            break;
+        case 5:
+            children = '★★★★★'
+            break;
+
+    }
```

Comment:
This works but it's very manual. You could do it in different ways such as a loop or string.repeat(). Also, you are reassigning the value that you get from the props